tools/repl_test.py

```diff
@@ -86,6 +86,19 @@ def test_multiline(self):
         assertEqual(err, '')
         assertEqual(code, 0)
 
+    def test_set_timeout(self):
+        # Special treatment
+        p = Popen([self.deno_exe], stdout=PIPE, stderr=PIPE, stdin=PIPE)
+        # Print after 1 second
+        p.stdin.write(
+            "setTimeout(() => console.log('HI'), 1000)\n".encode("utf-8"))
+        sleep(2)  # Wait 2 second before proceed
```

Comment:
Can you reduce the time here to 100 and 0.2 ? Just so it doesn't add much to test time.

Reply:
Done.